examples/multiphysics/rayleigh_benard_convection/rayleigh_benard_convection.prm

```diff
@@ -0,0 +1,166 @@
+  set number mesh adapt       		= 0
+  set output name             		= rayleigh-benard_convection
+  set output control               	= time
+  set output time			= 25
+  set output path                  	= ./output/
+end
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order        = 1
+    set pressure order        = 1
+    set temperature order     = 1
+end
+
+#---------------------------------------------------
+# Timer
+#---------------------------------------------------
+subsection timer
+    set type    = none                                  # <none|iteration|end>
+end
+
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+subsection initial conditions
+    set type 				= nodal
+    subsection uvwp
+            set Function expression 	= 0; 0; 0
+    end
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+  set number of fluids 			= 1
+  subsection fluid 0
+  	set density 			= 100
+  	set kinematic viscosity 	= 0.01
+  	set thermal expansion 		= 0.0002
+  	set thermal conductivity 	= 0.15625	# for Ra = 10000
+  	#set thermal conductivity 	= 0.0625	# for Ra = 25000
+  	set specific heat  		= 100
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+	set type 		= dealii
+	set grid type 		= subdivided_hyper_rectangle
+	set grid arguments 	= 2, 1: 0.5, 0.25 : 0 , 0 : true
+	set initial refinement 	= 6
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+subsection multiphysics
+	set buoyancy force = true
+	set heat transfer  = true
+	set fluid dynamics = true
+end
+
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 4
+    subsection bc 0
+    set id = 0
+        set type              = noslip
+    end
+    subsection bc 1
+    set id = 1
+        set type              = noslip
+    end
+    subsection bc 2
+    set id = 2
+        set type              = noslip
+    end
+    subsection bc 3
+    set id = 3
+        set type              = noslip
+    end
+end
+
+subsection boundary conditions heat transfer
+  set number                  = 4
+    subsection bc 0
+    set id = 0
+        set type	      = convection-radiation
+        set h	              = 0
+        set Tinf	      = 0
+        set emissivity        = 0
+    end
+    subsection bc 1
+    set id = 1
+        set type	      = convection-radiation
+        set h	              = 0
+        set Tinf	      = 0
+        set emissivity        = 0
```

Comment:
I like using this BC type, since it's a comprehensive BC in which I can define convection, radiation, combination of radiation-convection and insulation. :)

Reply:
No worries then.